recipes/newrelic/mongodb/debian.yml

```diff
@@ -0,0 +1,247 @@
+keywords:
+  - Infrastructure
+  - Integration
+  - mongodb
+
+# Examine Infrastructure events for correlated data
+processMatch:
+  - mongodb
+  - mongod
+
+# Matches partial list of the Log forwarding parameters
+# https://docs.newrelic.com/docs/logs/enable-log-management-new-relic/enable-log-monitoring-new-relic/forward-your-logs-using-infrastructure-agent#parameters
+logMatch:
+  - name: MongoDB log
+    file: /var/log/mongodb/mongod.log
+
+# NRQL the newrelic-cli will use to validate the agent/integration this recipe
+# installed is successfully sending data to New Relic
+validationNrql: "SELECT count(*) from MongoDatabaseSample where hostname like '{{.HOSTNAME}}' FACET entityGuid SINCE 10 minutes ago"
+
+# Prompts for input from the user. These variables then become
+# available to go-task in the form of {{.VAR_NAME}}
+inputVars:
+  - name: "NR_CLI_DB_USERNAME"
+    prompt: "Please enter your MongoDb username below."
+  - name: "NR_CLI_DB_CLUSTERNAME"
+    prompt: "Please enter your MongoDb culster name below."
+  - name: "NR_CLI_DB_PASSWORD"
+    prompt: "Please enter your MongoDb password below."
+    secret: true
+  - name: "NR_CLI_DB_AUTH"
+    prompt: "Please confirm auth source for MongoDb below."
+    default: "admin"
+  - name: "NR_CLI_DB_HOSTNAME"
+    prompt: "Please enter your MongoDb hostname below. If none is provided the default value: localhost will be used."
+    default: "localhost"
+  - name: "NR_CLI_DB_PORT"
+    prompt: "Please enter your MonogDb PORT below. If none is provided the default value: 27017 will be used."
+    default: 27017
+  - name: "NR_CLI_SSL"
+    prompt: "Are you using SSL? (y/n)"
+  - name: "NR_CLI_CERT_AUTH_FILE"
+    prompt: "Please enter the path to your Certificate Authority file."
+  - name: "NR_CLI_SKIP_SSL_VERIFY"
+    prompt: "Would you like to verify the certificate sent by the host? (y/n)"
+  - name: "NR_CLI_CLIENT_CERT_FILE"
+    prompt: "Please enter the path to your Client Certificate file, if required."
+  - name: "NR_CLI_CERT_PASSPHRASE"
+    prompt: "Please enter your passphrase to decrypt the PEM key file, if required."
+
+install:
+
+  version: "3"
+  silent: true
+
+  tasks:
+    default:
+      cmds:
+        - task: create_user_notification
+        - task: setup
+        - task: restart
+        
+    create_user_notification:
+      cmds:
+        - |
+          echo "_ Note: These steps must be completed in order for the MongoDB integration to function properly.
+          In the MongoDB shell, enter use admin.
+          Use the following command to create the listCollections role.
+          db.createRole({role: "listCollections", privileges: [{ resource: {db:"",collection:""}, actions: ["listCollections"] }], roles: []})
+          Use the following commands to create a new user, and assign clusterMonitor and listCollections roles to the user:
+          db.createUser({ user: "username", pwd: "password", roles: [ "clusterMonitor", "listCollections" ]})"
```

Comment:
@devfreddy / @Julien4218 Would we want to show this info like this? I'm not sure if it aligns with the new MVP mockups to put this here or have the CLI use the `preInstall` prompt to do this in a controlled manner. Thoughts?